README.md

```diff
@@ -16,6 +16,40 @@ minimal requirements with respect to application structure, routes etc. With
 its pluggable strategies it __can support all kinds of authentication and
 authorization mechanisms__.
 
+# Table of Contents
+
+**Information**
+
+* [What does it do?](#what-does-it-do)
+* [How does it work?](#how-does-it-work)
+* [Working example](#how-do-i-use-it)
```

Comment:
I think we should rename the header to _"Example app"_ or so well. _"How do I use it?"_ doesn't seem like a great header for that…